src/pages/News/News.js

```diff
@@ -1,20 +1,30 @@
 import React, { useEffect, useState } from "react";
 import { loadNews } from "../../api/news";
+import { Loader } from "../../components/Loader";
 
 export const News = () => {
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(false);
   const [news, setNews] = useState([]);
 
-  useEffect(async () => {
-    setIsLoading(true);
-    try {
-      const n = await loadNews();
-      setNews(n);
-    } catch (err) {
-      setError(err);
-    }
-    setIsLoading(false);
+  useEffect(() => {
+    (async function () {
```

Comment:
no need to such complex construction here
simply move all the function to separate like `const getNews = async () => {}` 
and call it inside `useEffect`